pkg/operator/operator.go

```diff
@@ -416,6 +434,53 @@ func (o *Operator) applyStatusInformers(a *apptypes.App, sequence int64, kotsKin
 	return nil
 }
 
+func (o *Operator) ResumeHelmStatusInformers() {
+	helmApps := helm.GetCache()
+	for {
+		helmApps = helm.GetCache()
```

Comment:
Can this be cleaned up somehow.  There is already a channel that triggers this function.  Maybe change initialization order?